frontend/views/UserProfile.vue

```diff
@@ -122,12 +122,12 @@ main
               | The Receiving Limit must be a numeric currency amount and may contain 2 decimal points.
 
       .has-text-centered.button-box
-        button.is-success(
+        i18n.is-success(
+          tag='button'
           v-if='currentGroupContractId'
           :disabled='$v.editedGroupProfile.$invalid'
           data-test='GroupProfileSubmitBtn'
-        )
-          i18n Save Group Profile
+        ) Save Group Profile
```

Comment:
So all `i18n` tags must be on a single line in order for the parser to easily extract the strings from them. So these properties all should be on one line (line 125), and "Save Group Profile" should also be on line 125.

This issue shows up in several other places in this commit/PR.

Reply:
That parser. Wouldn't it be easier to improve it, so we don't deteriorate the code quality across all the codebase? It's still a lot of places to change back.

Reply:
@sandrina-p TBH, no, it would be more difficult to improve it, but we can decide what to do about this later.

Reply:
Okay then :)